app/controllers/comments_controller.rb

```diff
@@ -33,12 +33,11 @@ def create
 
   def update
     @comment = Comment.find params[:id]
-
     if logged_in? && current_user.can_edit?(@comment)
-      Comment.update(@comment.id, :body => params[:comment][:body])
```

Comment:
This line was not updating the comment and the tests were failing. That's why I had to change it